lib/live_toast/live_component.ex

```diff
@@ -37,9 +37,13 @@ defmodule LiveToast.LiveComponent do
 
   @impl Phoenix.LiveComponent
   def render(assigns) do
+    assigns = assigns
+    |> assign_new(:id, fn -> "toast-group" end)
+    |> assign_new(:show_client_and_server_flashes, fn -> true end)
+
     ~H"""
-    <div id={assigns[:id] || "toast-group"} class={@group_class_fn.(assigns)}>
-      <div class="contents" id="toast-group-stream" phx-update="stream">
+    <div id={@id} class={@group_class_fn.(assigns)}>
+      <div class="contents" id={@id <> "-stream"} phx-update="stream" data-role="toast-group-stream">
```

Comment:
This is the important bit.
Without being able to change this HTML ID, we get duplicate HTML IDs when we have 2 toast groups rendered, e.g. when doing `live_render`